test/test_docker_api_language_def_json.py

```diff
@@ -0,0 +1,162 @@
+import json
+import shutil
+import unittest
+from pathlib import Path
+from tempfile import TemporaryDirectory, tempdir
+
+import docker  # type: ignore
+import pydantic_core
+import utils as exaslct_utils  # type: ignore # pylint: disable=import-error
+from exasol_integration_test_docker_environment.lib.docker.images.image_info import (
+    ImageInfo,
+)
+from exasol_integration_test_docker_environment.testing import utils  # type: ignore
+
+from exasol.slc.api import build
+from exasol.slc.internal.utils.docker_utils import find_images_by_tag
+from exasol.slc.models.language_definition_common import (
+    SLCLanguage,
+    UdfClientRelativePath,
+)
+from exasol.slc.models.language_definition_model import (
+    LanguageDefinition,
+    LanguageDefinitionsModel,
+)
+
+
+class ApiDockerBuildLangDefJsonTest(unittest.TestCase):
```

Comment:
I think, before we add the next feature, we maybe should add a pytest setup, such that every new test can use pytest

Reply:
Created #247 to convert all tests to pytest